postgres/README.md

```diff
@@ -0,0 +1,226 @@
+## PostgreSQL Example Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+To use HarbourBridge on a PostgreSQL database called mydb using pgdump output, run:
+
+```sh
+pg_dump mydb | harbourbridge -driver=pgdump
```

Comment:
Quick question: for consistency should we use "pg_dump" for the driver name (i.e. change the PGDUMP constant in main.go).
I don't feel too strongly, but the inconsistency kind of stands out in the README.
And if we want to make this change we should do it now.

Reply:
Yes agreed. The command that is being used by PostgreSQL is `pg_dump` and driver that we have used is `pgdump` and hence the inconsistency. If we change in the code right now, then all functionality of postgres needs to be tested once after changing it.

I also don't feel too strongly but sure it would be better from consistency point of view. Let me know if you think that we should change it. 

One suggestion - If you think that we should change `pgdump-->pg_dump`, then it would be a better idea to do a separate PR for that change and keep this PR focused on documentation.

Reply:
Yes, I think that's a good idea.
Let's do it in a separate PR.